### PR TITLE
create controller spec

### PR DESCRIPTION
**Pull request by Ntare22**
install rspec in project
create rspec files to test controllers

### CODE REVIEW

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Ntare22, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Add RSpec as a gem to your project.
- Correct Gitflow
- No linter error

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] As required, Kindly test for User and Post controllers `index` and `show` actions using RSpec
 - Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: test/controllers/pages_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  test 'should get hello' do
+    get pages_hello_url
+    assert_response :success
+  end
+end
```
Comment (andersonlebon) [inline test/controllers/pages_controller_test.rb]: - [ ] Kindly address issues mentioned by the previous reviewer. You need to have as directory rspec called requests that will contain all the tests you will write(`rspec/requests`)

## [OPTIONAL]
- [ ]  Also You need to include linters in your project currently I can't see any of them

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Ntare22 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉🍾**
Hello @Ntare22 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**